core/blockchain.go

```diff
@@ -2731,3 +2731,17 @@ func (bc *BlockChain) SubscribeBlockProcessingEvent(ch chan<- bool) event.Subscr
 func (bc *BlockChain) SupportsMultitenancy(context.Context) (*proto.PreAuthenticatedAuthenticationToken, bool) {
 	return nil, bc.isMultitenant
 }
+
+// Quorum
+
+// PopulateSetPrivateState function pointer for updating private state
```

Comment:
what this has to do with the `fix linter` commit?

Reply:
the merge was missing them with a conflict. I put it at the end so that will less conflict in future
https://github.com/ConsenSys/quorum/blob/master/core/blockchain.go#L224

Reply:
👍